postcss.config.js

```diff
@@ -6,7 +6,8 @@ module.exports = {
       addModulesDirectories: ['node_modules'],
     }),
     require('postcss-mixins'),
-    require('postcss-nested'),
     require('postcss-simple-vars')({ variables: breakpoints }),
+    require('tailwindcss/nesting'),
```

Comment:
tailwind's rec for nesting: https://tailwindcss.com/docs/using-with-preprocessors#nesting. it uses postcss-nested under the hood